File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/Embed.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class Embed{

	@SerializedName("width")
	private int width;

	@SerializedName("flashUrl")
	private String flashUrl;

	@SerializedName("flashSecureUrl")
	private String flashSecureUrl;

	@SerializedName("iframeUrl")
	private String iframeUrl;

	@SerializedName("height")
	private int height;

	public void setWidth(int width){
		this.width = width;
	}

	public int getWidth(){
		return width;
	}

	public void setFlashUrl(String flashUrl){
		this.flashUrl = flashUrl;
	}

	public String getFlashUrl(){
		return flashUrl;
	}

	public void setFlashSecureUrl(String flashSecureUrl){
		this.flashSecureUrl = flashSecureUrl;
	}

	public String getFlashSecureUrl(){
		return flashSecureUrl;
	}

	public void setIframeUrl(String iframeUrl){
		this.iframeUrl = iframeUrl;
	}

	public String getIframeUrl(){
		return iframeUrl;
	}

	public void setHeight(int height){
		this.height = height;
	}

	public int getHeight(){
		return height;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Embed.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class Embed : Parcelable, Serializable {
    @SerializedName("width")
    var width = 0

    @SerializedName("flashUrl")
    var flashUrl: String? = null

    @SerializedName("flashSecureUrl")
    var flashSecureUrl: String? = null

    @SerializedName("iframeUrl")
    var iframeUrl: String? = null

    @SerializedName("height")
    var height = 0
    override fun toString(): String {
        return "Embed{" +
                "width = '" + width + '\'' +
                ",flashUrl = '" + flashUrl + '\'' +
                ",flashSecureUrl = '" + flashSecureUrl + '\'' +
                ",iframeUrl = '" + iframeUrl + '\'' +
                ",height = '" + height + '\'' +
                "}"
    }
}